doc/nrf/index.rst

```diff
@@ -11,12 +11,13 @@ The |NCS| includes the Zephyr™ real-time operating system (RTOS), which is bui
 .. note::
 
    This version of the |NCS| supports prototyping and evaluation, but should not be used for production and deployment in end products.
-
-.. previous note about development and production support
-..   nRF53 Series devices (which are pre-production) and Zigbee and Bluetooth mesh protocols are supported for development in |release| for prototyping and evaluation.
```

Comment:
This part of the note has changed on master.
We'll need to figure out how we want to handle this note - should we have the note for the latest release on master, or should master always say "no production support"?
That decision will impact your addition as well, because the new sentence should only be displayed for the master branch.

Reply:
Eirik Midttun says: " I suggest leaving the nRF53 comment until it has Bluetooth production support (v1.6 at the latest)."